packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -485,10 +493,18 @@ const renderer =
                     new CopyPlugin({
                         patterns: [
                             {
-                                from: `${
-                                    EXT_OVERRIDES_DIR ? EXT_OVERRIDES_DIR_NO_SLASH + '/' : ''
-                                }app/static`,
-                                to: 'static/',
+                                from: path.posix.join(
```

Comment:
curious why are we using `path.posix`? my understanding is that we should not be tied to a specific platform and should be using `path.join` which works on both windows and unix?

Reply:
I had this in place during some debugging, good catch, this needs to roll back (Unix pathing is **correct**!)